schema/accounts.js

```diff
@@ -53,17 +27,22 @@ module.exports = {
 			publicKey: {
 				type: 'string',
 				format: 'publicKey'
-			}
-		}
-	},
-	top: {
-		id: 'accounts.top',
-		type: 'object',
-		properties: {
+			},
+			secondPublicKey: {
+				type: 'string',
+				format: 'publicKey'
+			},
+			username: {
+				type: 'string',
+				format: 'username'
+			},
+			orderBy: {
+				type: 'string'
+			},
 			limit: {
 				type: 'integer',
-				minimum: 0,
-				maximum: 100
+				minimum: 1,
+				maximum: 1000
```

Comment:
That's a significant change. @Isabello?

Reply:
I am changing this to `constants.activeDelegates`. That makes the most sense to me right now. Please correct me if you think something else is better.